apps/include/opt.h

```diff
@@ -38,6 +39,8 @@
         { "policy", OPT_V_POLICY, 's', "adds policy to the acceptable policy set"}, \
         { "purpose", OPT_V_PURPOSE, 's', \
             "certificate chain purpose"}, \
+        { "eku", OPT_V_EKU, 's', \
```

Comment:
Perhaps the option name should be a bit more descriptive:
```suggestion
        { "required-eku", OPT_V_EKU, 's', \
```
If you disagree, I won't insist.

Reply:
Well, most of the OpenSSL CLI option names are pretty short and not fully self-explanatory.
I believe that in the given context `-eku` should be clear enough, and it is fully in line with `-purpose`.

(And if we go for a longer name like the one suggested above, for consistency with others it should involve a `_` rather than a `-`.)